src/components/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,45 @@
+import loginSignupImage from "../../../assets/login-signup.png";
```

Comment:
and there's no need to import the image. 
We usually keep images in the public folder, in an images folder.
We can then just pass the relative url to the image like: 

```<img src="/images/myImage.png" />```

Reply:
Fixed the problem in the new commit.